### ISSUE TITLE
Linting styled-components still does not work

### ISSUE THREAD

**Issue by mssngr** [shinnn/vscode-stylelint#123]
Despite the addition of the `stylelint.additionalDocumentSelectors` config, linting css in `styled-components` still does not work.

Running stylelint from the command line finds the missing semicolon in this:
```js
export const Container = styled.div`
  display: flex
  flex-wrap: nowrap;
  justify-content: space-between;
  align-items: center;
`
```
![screenshot 2017-12-06 10 24 10](https://user-images.githubusercontent.com/23464169/33675399-a481f8e4-da6f-11e7-9e09-4777c5611c6b.png)

But the editor does not:
![screenshot 2017-12-06 10 23 28](https://user-images.githubusercontent.com/23464169/33675360-858bae9e-da6f-11e7-9afe-f7b34a5cba6e.png)

Everything is installed correctly, and below are my config files.

```json
// In my VSCode user settings...
    "stylelint.enable": true,
    "stylelint.additionalDocumentSelectors": [
        "js",
        "jsx",
        "ts",
        "vue"
    ],
    "css.validate": false,
    "scss.validate": false
```
```json
// In my package.json
    "stylelint": "^8.3.1",
    "stylelint-config-standard": "^18.0.0",
    "stylelint-config-styled-components": "^0.1.1",
    "stylelint-processor-styled-components": "^1.2.0"
```
```json
// My .stylelintrc
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": [
    "stylelint-config-standard",
    "stylelint-config-styled-components"
  ],
  "syntax": "scss"
}
```

**Comment by shinnn** [comment]
1. Probably you're using the VSCode built-in "JavaScript" language mode for your `styles.js`.
2. The language ID of the "JavaScript" mode is not `js`. Check [the document about VSCode language identifiers](https://code.visualstudio.com/docs/languages/identifiers).
